bin/run_ccm_performance_test.sh

```diff
@@ -92,6 +86,14 @@ set_yaml_based_filter() {
   done
 }
 
+create_cluster() {
+  ccm create -n 1 -v ${CLUSTER_VERSION} ecaudit
```

Comment:
maybe pass an argument (same as to run_stress) and give that to the cluster-name ->
ccm create -n 1 -v ${CLUSTER_VERSION} ecaudit-${1}

That way it would be "simple" to update the stop_cassandra() function:
ccm remove -> ccm stop
IF you would want to keep the cluster-data between the tests